doc/build/build_options.md

```diff
@@ -114,6 +115,16 @@ TBB plus more:
   primitive creation time. At the primitive execution time, the threadpool is
   responsible for balancing the static decomposition from the previous item
   across available worker threads.
+  
+#### Vendor BLAS libraries
+oneDNN can use a standard BLAS library for GEMM operations. The `DNNL_BLAS_VENDOR` build option controls BLAS library selection, and defaults to `NONE`. For AArch64 builds with GCC, use the [Arm Performance Libaries](https://developer.arm.com/tools-and-software/server-and-hpc/downloads/arm-performance-libraries):
```

Comment:
nit (here, above, and below): please try to fit into 80 symbols (unless really required, e.g. due to tables and urls)